config/shiny-server-rules.config

```diff
@@ -189,3 +189,9 @@ preserve_logs {
   maxcount 1;
 }
 
+log_as_user {
```

Comment:
We'll want to update the admin guide as well. Probably reference from `user_dirs`